ajax/dropdownAllItems.php

```diff
@@ -54,7 +54,7 @@
         $link = "getDropdownUsers.php";
     }
 
-    $rand = $_POST['rand'] ?? mt_rand();
+    $rand = htmlescape($_POST['rand'] ?? mt_rand());
```

Comment:
We could consider that random suffix is always an integer.

```suggestion
    $rand = (int) ($_POST['rand'] ?? mt_rand());
```